mobile/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -19,27 +21,45 @@ const SignUpForm: FC<Props> = ({ onSubmit }) => {
   });
 
   return (
-    <>
-      <Text>Sign Up</Text>
+    <View style={styles.container}>
+      <Image source={require('../../../../assets/images/logo.png')} />
```

Comment:
instead of require use Image.resolveAssetSource (create a helper for this), also add height for this component in styles